MapboxNavigation/RouteMapViewController.swift

```diff
@@ -268,6 +270,56 @@ extension RouteMapViewController: NavigationMapViewDelegate {
             return defaultReturn
         }
         
+        if let userLocation = mapView.userLocation,
+            let style = mapView.style,
+            routeController.showCurrentWayNameLabel == true {
```

Comment:
If `showCurrentWayNameLabel` is a Boolean, is it necessary to compare it to `true`?